src/main/java/de/tum/in/www1/artemis/repository/plagiarism/PlagiarismCaseRepository.java

```diff
@@ -35,6 +36,15 @@ public interface PlagiarismCaseRepository extends JpaRepository<PlagiarismCase,
             """)
     List<PlagiarismCase> findByCourseIdWithPlagiarismSubmissionsAndComparison(@Param("courseId") Long courseId);
 
+    @Query("""
+            SELECT DISTINCT plagiarismCase FROM PlagiarismCase plagiarismCase
+            LEFT JOIN FETCH plagiarismCase.post
+            LEFT JOIN FETCH plagiarismCase.plagiarismSubmissions plagiarismSubmissions
+            LEFT JOIN FETCH plagiarismSubmissions.plagiarismComparison plagiarismComparison
+            WHERE plagiarismCase.exercise.exerciseGroup.exam.id = :examId
+            """)
+    List<PlagiarismCase> findByExamIdWithPlagiarismSubmissionsAndComparison(@Param("examId") Long examId);
```

Comment:
:bulb: Add `Post` to method name as well

Reply:
The version of this method for Course ([just above this one](https://github.com/ls1intum/Artemis/pull/5620/files/3671d7574d8d9c3a514af403d21939aaea4a6712#diff-30868ed17409051376cbb46d6aade4d5a85f41b6661b8b7a9e5d0483dd3e3f8cR30-R37)) doesn't have `Post` in its name either so I wanted to keep them similar. Should we change both of them instead?